command/commands/user.py

```diff
@@ -155,6 +157,10 @@ def edit_helper(self, user_id, param_list):
             edited_user.set_position(param_list["pos"])
         if param_list["github"]:
             edited_user.set_github_username(param_list["github"])
+            try:
+                self.github.org_add_member(param_list["github"])
+            except GithubAPIException as e:
+                logging.error("Error editing github organization!\n")
```

Comment:
I would change the error message to:

```python
logging.error("Error adding user %s to GitHub organization" % param_list['github'])
```

Also, we should tell the user about the error. It should be appended to the `User edited` thing as a new line or something. Just for some feedback to the user.

Also, you may want to place line 159 after line 161 since we may not want to add a GitHub username to the model if we can't even add it to the GitHub organization.